content/en/dashboards/list/_index.md

```diff
@@ -0,0 +1,95 @@
+
+To edit the teams associated with one or more dashboards, take the following steps:
+1. Select the checkbox next to each dashboard you wish to modify.
+1. Open the **Edit Teams** dropdown in the upper right.
+1. Use the checkboxes to select the appropriate teams for the dashboards.
+1. Click **Apply Changes**.
+
+## Lists
+
+Dashboard lists groups dashboards so you and your team can switch between dashboards within the same context. You can add dashboards to [preset lists](#preset-lists) or to a custom list.
+
+1. To create a dashboard list, click **+ New List** in the upper right.
+1. Click the pencil icon to change a list's title. The list's title is automatically set with the user's first name. For example, `John's list`. 
+1. Add dashboards to a list. In the **[All Dashboards](#view-all-dashboards)** table, check the check boxes next to the Dashboard title. Then click the **Add to** dropdown in the upper right corner of the Dashboard list and select the list.
+
+The left sidebar displays all lists, which you can filter by Team or through search terms. Toggle **Hide Controls** to hide this sidebar.
+
+### Favorite lists
+
+Favorite lists are dashboard lists starred by the current logged in user. **Note**: If you have no starred lists, the *Favorite Lists* category is hidden.
+
+### Preset lists
+
+Preset lists are out-of-the-box dashboard lists in Datadog:
+
+| List                     | Description                                                               |
+|--------------------------|---------------------------------------------------------------------------|
+| All Custom               | Custom dashboards made by any team member in your organization's account. |
+| All Hosts                | Automatic dashboards created by Datadog when you add a host.              |
+| All Integrations         | Automatic dashboards created by Datadog when you install an integration.  |
+| All Shared               | Dashboards with authenticated or public link sharing enabled.             |
+| Created By You           | Custom dashboards created by the current user.                            |
+| Frequently Viewed By You | All dashboards frequently viewed by the current user.                     |
+| Recently Deleted         | Dashboards deleted within the last 30 days. [Restore deleted dashboards](#restore-deleted-dashboards) from this list.|
+| Security and Compliance  | Out-of-the-box Security dashboards.                                       |
+
+### Restore deleted dashboards
+
+Use the preset **Recently Deleted** list to restore deleted dashboards. From the list, select all dashboards to restore and click **Restore to**. Select a specific list to restore the dashboards to, or select **All Custom** to restore them without a custom list. Dashboards in **Recently Deleted** are permanently deleted after 30 days.
+
+{{< img src="dashboards/recently_deleted.png" alt="Restore deleted dashboard" style="width:100%;">}}
```

Comment:
I know you didn't add the screenshot, but I noticed it has minor PII in it (someone's avatar), just wanted to flag!